docs/source/data_acquisition/DataProv-MSDefender.rst

```diff
@@ -87,6 +88,45 @@ Loading a QueryProvider for M365 Defender
 
 You can also use the aliases "MDE" and "MDATP".
 
+Specifying an API to Connect to
```

Comment:
Something like 
"Specifying the Defender Cloud Instance to Connect to"
Might be more visible in the docs.